src/Keboola/S3Extractor/Config.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Keboola\S3Extractor;
+
+use Keboola\Component\Config\BaseConfig;
+
+class Config extends BaseConfig
+{
+    /**
+     * @return string
+     */
+    public function key(): string
```

Comment:
Nedělám review, jen jsem šel kolem. ;o) My u getterů AFAIK všude dáváme prefix `get` (`getKey`), tak to ještě prosím tě uprav ať je to konzistentní.

Reply:
Já prefix používám jen když je to třeba, ale oka, doplním a budu na to příště myslet. Díky! :)